pkg/config/config.go

```diff
@@ -631,6 +631,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("container_env_as_tags", map[string]string{})
 	config.BindEnvAndSetDefault("container_labels_as_tags", map[string]string{})
 
+	// Podman
+	config.BindEnvAndSetDefault("podman_db_path", "/var/lib/containers/storage/libpod/bolt_state.db")
```

Comment:
please reuse the "const string" `podman.DefaultDBPath` to set the default value.

Reply:
Fix it! https://github.com/DataDog/datadog-agent/pull/16420/commits/34d77d964274af528b1c1dcc7eb154ab7c1fd9d9

Reply:
You cannot remove the build tag, it's necessary so you cannot use `podman.DefaultDBPath` here, but with the default value being set here, we don't need `podman.DefaultDBPath` anymore.